test/README.md

```diff
@@ -28,9 +28,10 @@ More complex tests may require different inputs, but the expected values should
 The jenkins job executes as follows: 
 ```bash
 VAULT_TOKEN=$(cat /etc/vault-token-dsde)
-INPUTS_JSON="https://raw.githubusercontent.com/HumanCellAtlas/skylab/master/test/optimus/pr/test_inputs.json"
-WDL_FILE="/working/test/optimus/pr/TestOptimusPR.wdl"
-DEPENDENCIES_JSON="/working/test/optimus/pr/dependencies.json"
+PIPELINE_FOLDER_NAME=<optimus|smartseq2_single_sample>
+INPUTS_JSON="https://raw.githubusercontent.com/HumanCellAtlas/skylab/master/test/${PIPELINE_FOLDER_NAME}/pr/test_inputs.json"
+WDL_FILE="/working/test/${PIPELINE_FOLDER_NAME}/pr/Test${PIPELINE_FOLDER_NAME}PR.wdl"
+DEPENDENCIES_JSON="/working/test/${PIPELINE_FOLDER_NAME}/pr/dependencies.json"
```

Comment:
Are we still using a dependencies.json file for the tests?

Reply:
We are. Do you have a suggestion to remove them?

Reply:
Oh, I'm thinking of how lira isn't using the dependencies.json file to run the adapter wdls anymore. Nevermind!